job-shop-scheduling/README.rst

```diff
@@ -1,8 +1,91 @@
-Demo of Job Shop Scheduling
-===========================
+Job Shop Scheduling Demo
+========================
+A demo on how to optimally schedule jobs using a quantum computer.
 
-Further Information
--------------------
+Given a set of jobs and a finite number of machines, how should you schedule
+your jobs on those machines such that all your jobs are completed at the
+earliest possible time? This question is the Job Shop Scheduling Problem!
+
+Each of our jobs can be broken down into smaller machine-specific tasks. For
+example, the job of making pancakes can be broken down into several
+machine-specific tasks: mixing ingredients in a *mixer* and cooking the batter
+on the *stove*. There is an order to these tasks (ex. you can't bake the batter
+before you mix the ingredients) and there is a time associated with each task
+(ex. 5 minutes on the mixer, 2 minutes to cook on the stove). Now supposing
```

Comment:
ex. --> e.g.,

Reply:
"Now supposing..." the previous paragraph already defined that you have "a finite number of machines" so I'd suggest changing here to "Given that you have..." for consistency